app/exporters/patients_exporter.rb

```diff
@@ -51,6 +52,10 @@ def self.csv_headers
       "Patient District",
       (zone_column if Rails.application.config.country[:patient_line_list_show_zone]),
       "Patient State",
+      "Assigned Facility Name",
```

Comment:
@prabhanshuguptagit @harimohanraj89 Do we want to double-check if this should be called Preferred Facility? (not a blocker)

Reply:
Good question. We shall check.